encord/objects/ontology_labels_impl.py

```diff
@@ -499,8 +499,20 @@ def get_annotation(self, frame: Union[int, str] = 0) -> Annotation:
                 Defaults to the first frame.
         """
         if isinstance(frame, str):
-            frame = self._parent.get_frame_number(frame)
-        return self.Annotation(self, frame)
+            # TODO: this check should be consistent for both string and integer frames,
```

Comment:
I'm not sure whether this comment is helpful enough. I don't know what part of the parsing logic prevents to do the to do